aptos-move/aptos-vm/src/natives.rs

```diff
@@ -58,6 +58,22 @@ pub fn aptos_natives(
         .collect()
 }
 
+pub fn aptos_natives_abstract_usage(builder: &mut SafeNativeBuilder) -> NativeFunctionTable {
```

Comment:
Let's rename this to `aptos_natives_with_builder` and make `aptos_natives` call it so to get rid of the duplicate code.